types/next-redux-wrapper/index.d.ts

```diff
@@ -49,7 +49,7 @@ declare namespace nextReduxWrapper {
         createStore: NextStoreCreator<TInitialState, TStateProps, TDispatchProps, TOwnProps, TMergedProps>;
         debug?: boolean;
         storeKey?: string;
-        mapStateToProps?: MapStateToPropsParam<TStateProps, TOwnProps>;
+        mapStateToProps?: MapStateToPropsParam<TStateProps, TOwnProps, TMergedProps>;
```

Comment:
Please help me understand the motivation for this change. Could you please add a test case, showing intended usage, that would fail without your changes?

Reply:
This is fix https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/react-redux/index.d.ts#L161  which MapStateToPropsParam required 3 arguments, compile will broken without this change.

Reply:
Thanks. Could you add a test case?